pkg/scorecard/statement.go

```diff
@@ -0,0 +1,83 @@
+// Copyright 2025 OpenSSF Scorecard Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Subject {
+// 	 Repo           jsonRepoV2          `json:"repo"`
+// }
```

Comment:
is this intentionally still in here? or leftover?

Reply:
Oops I didn't see that with the copyright boilerplate next to it.. Removed